src/components/Editor/tests/Editor.spec.js

```diff
@@ -30,10 +30,15 @@ function createMockEditor() {
       defaultTextHeight: () => 0,
       display: { gutters: { querySelector: jest.fn() } }
     },
+    editor: {
+      getLine: line => ""
+    },
```

Comment:
do we need this?

Reply:
No sorry, the merge ended up a little wonky, thanks for the double check, I'll remove it right now.